test/formatting.coffee

```diff
@@ -298,3 +298,10 @@ test "#1275: allow indentation before closing brackets", ->
     a = 1
    )
   eq 1, a
+
+test "#3199: throw multiline implicit object", ->
```

Comment:
@vendethiel ya I was just being lazy about trying to minimize merge conflicts once this is eventually merged into `2` with #4599 but you're right, updated